components/supervisor-api/terminal.proto

```diff
@@ -14,14 +14,21 @@ service TerminalService {
     // Open opens a new terminal running the login shell
     rpc Open(OpenTerminalRequest) returns (OpenTerminalResponse) {}
 
-    // Close closes a terminal for the given alias, SIGKILL'ing all child processes
+    // Shutdown closes a terminal for the given alias, SIGKILL'ing all child processes
     // before closing the pseudo-terminal.
-    rpc Close(CloseTerminalRequest) returns (CloseTerminalResponse) {
+    rpc Shutdown(CloseTerminalRequest) returns (CloseTerminalResponse) {
```

Comment:
`@grcp/grpc-js` already has `close` method in the base client and generated one will override. It was caught by fixing https://github.com/gitpod-io/gitpod/pull/2972/files#r563487965

Reply:
For consistencies sake we should also rename `CloseTerminalRequest`/`CloseTerminalResponse` to `ShutdownTerminalRequest`/`ShutdownTerminalResponse`